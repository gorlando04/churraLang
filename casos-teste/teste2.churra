amigo "A" contribui 120
amigo "B" contribui 80
item bebida preco_L 20 quantidade_L 5
rateio igual