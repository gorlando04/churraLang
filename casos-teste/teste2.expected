Total do churrasco: R$100.00
A deve pagar: R$50.00
B deve pagar: R$50.00